addons/im_livechat/tests/test_get_mail_channel.py

```diff
@@ -46,6 +51,9 @@ def test_get_mail_channel(self):
             channel_operator_ids = [channel_operator[0] for channel_operator in channel_operators]
             self.assertTrue(all(partner_id in channel_operator_ids for partner_id in self.operators.mapped('partner_id').ids))
 
+        visitor_user_channel = self.livechat_channel._get_mail_channel('Visitor', user_id=self.visitor_user.id)
+        self.assertEqual(visitor_user_channel['correspondent_name'], self.visitor_user.name, "Chat title is incorrect.")
```

Comment:
Shouldn't you test `visitor_user_channel['name']` here, instead of `visitor_user_channel['correspondent_name']` as it's `name` that you set in https://github.com/odoo/odoo/pull/33831/files#diff-bf9431d0defc303ffbbe25345b806bf3R178

Reply:
`visitor_user_channel['name']` will be the name of the channel which is displayed on the visitor side and `visitor_user_channel['correspondent_name']` will be the name displayed to the operator while chatting. So I think it's better to test in `visitor_user_channel['correspondent_name']`.
Let me know your thoughts.

Reply:
Ah ok, perfect then.